fullscreen.bs

```diff
@@ -209,7 +219,22 @@ if all of the following are true, and false otherwise:
  <!-- cross-process, recursive -->
 </ul>
 
-<p>The <dfn method for=Element><code>requestFullscreen()</code></dfn> method, when invoked, must run
+<p>The {{FullscreenOptions}} dictionary can be used to help customize the User Agent UI:
```

Comment:
I suspect @foolip will comment on this as well, but this should be defined as part of the method definition.